apps/meteor/app/api/server/v1/presence.ts

```diff
@@ -14,6 +14,18 @@ API.v1.addRoute(
 	},
 );
 
+API.v1.addRoute(
+	'presence.getMonthlyPeakConnections',
```

Comment:
instead of creating the endpoint why not just get the IStats as we are doing with other info?

Reply:
hmmm I thought an endpoint would fit better in this case since we don't want just the raw value that is in the statistics (the daily peak). Instead, we want the maximum daily peak from the last 30 days (+ consider the current peak from the ongoing day so that this info makes more sense) 🤔 
Do you think it'd be enough to grab this from the statistics?

Reply:
I changed this to add the monthly data into the stats as well